C: Basic Output documentation update (#1031)
Codecademy/docs: content/c/concepts/basic-output/basic-output.md

---
Title: 'Basic Output'

> Description: 'In the C lang the console output is managed by the library stdio.h which has to be included.

Reviewer: ```suggestion
Description: 'In the C language the console output is managed by the library stdio.h which has to be included.
```

> Thanks to that you can use function like putchar() that target the C console.'

Reviewer: ```suggestion
Thanks to that a functions that target the C console, like putchar(), can be used.'
```

Reviewer: We should avoid "you" and "we" in docs.

  - 'paths/computer-science'
---

In the C language, **basic console output** is achieved using functions from the "standard input output" header file (**stdio.h**).

## Header File

C itself has no ability to output any information. In order to so, the **stdio.h** header file must be included in the program, which commonly happens at the top of the file:

```c
#include <stdio.h>
```

Once this is done, the functions included in it can be used.

> ## Format Specifiers

Reviewer: The Format Specifier & Decimal Precision sections should go with the new printf() term now.

Reviewer: These still need to be removed from this file.
